src/analyzer.js

```diff
@@ -179,24 +183,22 @@ function readStatsFromFile(filename) {
 }
 
 function getChildAssetBundles(bundleStats, assetName) {
-  return (bundleStats.children || []).find((c) =>
-    _(c.assetsByChunkName)
-      .values()
-      .flatten()
-      .includes(assetName)
-  );
+  return flatten(
+    (bundleStats.children || [])
+      .find((c) => Object.values(c.assetsByChunkName))
+  )
+    .includes(assetName);
 }
 
 function getBundleModules(bundleStats) {
-  return _(bundleStats.chunks)
-    .map('modules')
-    .concat(bundleStats.modules)
-    .compact()
-    .flatten()
-    .uniqBy('id')
-    // Filtering out Webpack's runtime modules as they don't have ids and can't be parsed (introduced in Webpack 5)
```

Comment:
Please keep these documentation comments as they give reasoning why certain logic exists :)

Reply:
Ah! I keep forgetting this!